src/MagnumPlugins/AssimpImporter/Test/AssimpImporterTest.cpp

```diff
@@ -625,6 +627,38 @@ void AssimpImporterTest::materialMultipleTextures() {
     }
 }
 
+void AssimpImporterTest::materialCoordinateSets() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("AssimpImporter");
+    {
+        std::ostringstream out;
+        Error redirectError{&out};
+        CORRADE_VERIFY(importer->openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "material-coordinate-sets.dae")));
+
+        CORRADE_EXPECT_FAIL("Material is loaded since coordinate sets are not imported");
+        CORRADE_VERIFY(!importer->material(0));
+
+        CORRADE_COMPARE(out.str(), "Trade::AssimpImporter::material(): multiple texture coordinate sets are not allowed by default, enable allowMaterialTextureCoordinateSets to import them\n");
+    }
```

Comment:
As with TinyGltf, I would split the first part into a separate test, `materialTextureCoordinateSetsDefault()`